Crashlytics/Crashlytics/Models/Record/FIRCLSReportAdapter.m

```diff
@@ -26,6 +26,8 @@
 #import <nanopb/pb_decode.h>
 #import <nanopb/pb_encode.h>
 
+const NSUInteger IMPORTANCE_FOR_EXCEPTION = 0;
```

Comment:
The naming for this may be confusing since the importance for exceptions be non-zero. Take a look at Protobuf.scala:384.

Maybe this should be DEFAULT_IMPORTANCE_FOR_EXCEPTION?